engine/tests/vm38/test_dispatch.py

```
@@ -86,12 +87,23 @@ def test_all_phase_hooks_register_to_correct_tasks(self) -> None:
             (on_finalize,  "finalize_step"),
             (on_postreqs,  "get_postreqs"),
             (on_gather_ns, "gather_ns"),
+            (on_get_ns,    "gather_ns"),
         ]
         for on_hook, expected_task in pairs:
             on_hook(lambda *, caller, ctx, **kw: None)
```

Ruff (0.15.1)

[warning] Unused lambda argument: `caller`

(ARG005)